app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -529,7 +529,6 @@ function AcuantCapture(
     });
 
     setImageCaptureText('');
-    setIsCapturingEnvironment(true);
```

Comment:
This is not strictly related to this PR, but it might cause other problems so I'm removing.